project_purchase_link/models/project_project.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2019 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import _, api, fields, models
```

Comment:
```suggestion
from odoo import _, fields, models
```

Reply:
Thank you very much @emagdalenaC2i  for all of these answers but there is still a problem and i don't see which one.

when i run flake8 on my computer i get errors but i don't know what to do : 
```
$ flake8 project_purchase_link
project_purchase_link/__init__.py:4:1: F401 '.models' imported but unused
project_purchase_link/tests/__init__.py:4:1: F401 '.test_project_purchase_link' imported but unused
project_purchase_link/models/__init__.py:4:1: F401 '.project_project' imported but unused
```

Reply:
now the pre-commit works on my computer : 
```
13:25 $ pre-commit run -a
black....................................................................Passed
prettier.................................................................Passed
prettier xml plugin......................................................Passed
eslint...............................................(no files to check)Skipped
- hook id: eslint
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
Debug Statements (Python)................................................Passed
Fix python encoding pragma...............................................Passed
Check for case conflicts.................................................Passed
Check docstring is first.................................................Passed
Check that executables have shebangs.................(no files to check)Skipped
Check for merge conflicts................................................Passed
Check for broken symlinks............................(no files to check)Skipped
Check Xml................................................................Passed
Mixed line ending........................................................Passed
flake8 except __init__.py................................................Passed
flake8 only __init__.py..................................................Passed
pylint with optional checks..............................................Passed
- hook id: pylint
- duration: 2.29s
pylint with mandatory checks.............................................Passed
pyupgrade................................................................Passed
isort except __init__.py.................................................Passed
Generate default setup.py for an addons directory........................Passed
✔ ~/docker-compose/OCA/project_purchase_link-13/addons/project [13.0-mig-project_purchase_link|✔] 
```